### ISSUE TITLE
Spectrum display not starting when it should

### ISSUE THREAD

**Issue by kpreid** [kpreid/shinysdr#69]
(This is just to document a known issue I am working on tracking down.)

I am seeing an intermittent problem where, after the first client connects to a freshly started server, the __Spectrum__ view will not display any spectrum data until things are nudged by creating a receiver.

The problem seems to be somewhere in the path from `MonitorSink` reporting interest to `Top` deciding to start in response to that.

**Comment by kpreid** [comment]
Fixed in b72beca84be6d7c76303a8c0241da970c0710b07.

Cause: When the new cell subscription interface was introduced in c034715d74bbf295285feb8ead55eb4d01ae0a74, it passed an argument to callbacks where the old one did not. When I added a lambda to drop the unused-in-that-case argument for this particular subscription, I forgot the parentheses to actually call `Top.__start_or_stop_later` rather than merely returning it.
